src/parser/nmodl_driver.hpp

```diff
@@ -67,43 +60,58 @@ class NmodlParser;
 class NmodlDriver {
   private:
     /// all macro defined in the mod file
-    std::map<std::string, int> defined_var;
+    std::unordered_map<std::string, int> defined_var;
 
     /// enable debug output in the flex scanner
     bool trace_scanner = false;
 
     /// enable debug output in the bison parser
     bool trace_parser = false;
 
-    /// pointer to the lexer instance being used
-    NmodlLexer* lexer = nullptr;
-
-    /// pointer to the parser instance being used
-    NmodlParser* parser = nullptr;
```

Comment:
I no longer remember why these members were added. May be during initial testing when I wanted to access lexer/parser instance from driver? But if it's not used then good to remove ! :+1: